libs/index/odc/index/stac.py

```diff
@@ -102,10 +102,13 @@ def _get_stac_bands(
             }
 
         path = asset["href"]
+        band_index = asset["band"] if "band" in asset else None
```

Comment:
```suggestion
        band_index = asset.get("band", None)
```
Keep it simple!

Reply:
Thanks for the review. Yes, this makes sense. See the updated code.